frontend/src/hocs/withHostCommunication/types.ts

```diff
@@ -155,3 +155,8 @@ export type IGuestToHostMessage =
 export type VersionedMessage<Message> = {
   stCommVersion: number
 } & Message
+
+export type IAllowedMessageOriginsResponse = {
```

Comment:
Hm, seems like we're a bit inconsistent about it since a good number of the types defining the structure of JSON that we receive via `postMessage` in `withHostCommunication/types.ts` have the `I` prefix.

I'll keep this as-is for now so that things stay consistent within the file, but I wouldn't mind dropping it everywhere.